portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -47,16 +69,19 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
   public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
     String enteredName = request.getParameter("name-entry");
     String enteredComment = request.getParameter("comment-entry");
-    Comment comment = new Comment(enteredName, enteredComment);
-    //String json = convertToJson(comment);
-    comments.add(enteredComment); //testing to se if it's working
+    // block completely empty comments
+    if(!(enteredName.trim().equals("") && enteredComment.trim().equals(""))) {
+      long timestamp = System.currentTimeMillis();
+      // upload via datastore
+      Entity commentEntity = new Entity(commentEntityName);
+      commentEntity.setProperty(authorFieldName, enteredName);
```

Comment:
I was thinking about how to implement that and I'm not sure if there is a good way. The reason I don't just store them in the first place is that every time the comments page is refreshed it needs to pull the entities from the datastore to be up-to-date with all the latest comments, including those from other users. A possible solution would be to aggregate new comments to a stored list of comments, but that entire list would have to be compared to the list of comments being pulled from the database, at which point the servlet may as well just return the data pulled from the database instead of needing to compare it with stored data first. Do you have any ideas on how I could efficiently store comments?